website/identifiers/tasks.py

```diff
@@ -17,3 +17,29 @@ def task__update_doi_metadata_on_change(self, target_guid):
 @celery_app.task(ignore_results=True)
 def update_doi_metadata_on_change(target_guid):
     task__update_doi_metadata_on_change(target_guid)
+
+@celery_app.task(bind=True, max_retries=5, acks_late=True)
+def task__update_doi_metadata_with_verified_links(self, target_guid):
```

Comment:
The task misses one important data: which is the links and types sent from GV. The task expects them to come in as arguments and the task should update the node. Otherwise, identifier update and Datacite tree-walk will see an empty dict.

In addition, I am wondering if we can make identifier update automatic when this field is saved/updated on node.